BUILD.bazel

```diff
@@ -46,13 +46,13 @@ go_binary(
 
 # Create the non-root user, "bazelremote", so the container can run with reduced
 # privileges.
-BAZEL_REMOTE_USER = "bazelremote"
+BAZEL_REMOTE_USER_NAME = "bazelremote"
+BAZEL_REMOTE_USER_ID = 1000
 passwd_entry(
     name = "bazelremote_passwd_entry",
-    username = BAZEL_REMOTE_USER,
-    info = BAZEL_REMOTE_USER,
-    uid = 1000,
-    gid = 1000,
+    username = BAZEL_REMOTE_USER_NAME,
+    info = BAZEL_REMOTE_USER_NAME,
```

Comment:
Can we can skip this and just use the default value for "info"?

Reply:
Done